webpackConfig.js

```diff
@@ -138,6 +138,9 @@ module.exports = (env, argv) => {
         'node_modules',
         path.resolve(__dirname, 'node_modules'),
       ],
+      alias: {
+        "ditto-react": path.resolve(__dirname, "node_modules/@jordin/react-library-testing")
```

Comment:
This is a hack so that we can test the latest version of `react-ditto` that a PR is up for without having to go through the code and update all instances of `import xyz from 'react-ditto'` to `import xyz from '@jordin/react-library-testing'`